src/SprykerEco/Service/AkeneoPim/Api/Adapter/AbstractApiAdapter.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Service\AkeneoPim\Api\Adapter;
+
+use Akeneo\Pim\ApiClient\AkeneoPimClient;
+
+abstract class AbstractApiAdapter implements ApiAdapterInterface
```

Comment:
Shouldn't this class be implemented by the other classes in this directory? Currently there is no usage

Reply:
Looks like this abstract class is a redundant class here.